src/include/bitset.hpp

```diff
@@ -0,0 +1,72 @@
+
+#pragma once
+
+template <const size_t N>
+class bitset
+{
+
+private:
```

Comment:
I would put everything private after public, so that the first thing you see is the API.

Reply:
Done.